qunetsim/backends/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .cqc_backend import CQCBackend
+from .eqsn_backend import EQSNBackend
+from .projectq_backend import ProjectQBackend
+from .re_lock import RWLock
```

Comment:
Maybe the RWLock and SafeDict do not have to be exported, since a user should not access them.